src/classes/CRLP_RollupUI_SVC.cls

```diff
@@ -437,13 +437,15 @@ public with sharing class CRLP_RollupUI_SVC {
                 'edit' => Label.stgBtnEdit,
                 'error' => Label.PageMessagesError,
                 'field' => Schema.Filter_Rule__mdt.Field__c.getDescribe().getLabel(),
+                'filterGroupDeleteConfirm' => String.format(Label.CMT_DeleteConfirm, new List<String>{Schema.SObjectType.Filter_Group__mdt.getLabel()}),
```

Comment:
I like the reusability of CMT_DeleteConfirm, but we may need to use more specific error messages about the consequences of deleting each of these items. @smarovich made a suggestion for Delete Filter Rule warning in Rollups UI Text, and may be able to suggest text for the filter group and rollup warnings.

Reply:
discussed. leaving as-is for now.